huaweicloud/services/meeting/resource_huaweicloud_meeting_conferences.go

```diff
@@ -0,0 +1,1111 @@
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"user_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+
+			// Arguments
+			"topic": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 128),
+			},
+			"meeting_room_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"duration": {
+				Type:         schema.TypeInt,
+				Required:     true,
+				ValidateFunc: validation.IntBetween(15, 1440),
+			},
+			"start_time": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^\d{4}-\d{2}-\d{2} \d{2}:\d{2}$`),
+					"The start time format should be 'YYYY-MM-DD hh:mm'."),
+			},
+			"media_types": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					ValidateFunc: validation.StringInSlice([]string{
+						string(MediaTypeVoice), string(MediaTypeVideo), string(MediaTypeHDVideo),
+						string(MediaTypeData),
+					}, false),
+				},
+			},
+			"is_auto_record": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.IntInSlice([]int{
+					int(RecordModeOff), int(RecordModeOn),
+				}),
+			},
+			"encrypt_mode": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.IntInSlice([]int{
+					int(EncryptModeAdapt), int(EncryptModeForce), int(EncryptModeOff),
+				}),
+			},
+			"language": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(LanguageTypeZhCn), string(LanguageTypeEnUs),
+				}, false),
+			},
+			"timezone_id": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"record_type": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.IntInSlice([]int{
+					int(RecordTypeOff), int(RecordTypeLive), int(RecordTypeRc), int(RecordTypeLiveRc),
+				}),
+			},
+			"live_address": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"aux_address": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"is_record_aux_stream": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.IntInSlice([]int{
+					int(RecordModeOff), int(RecordModeOn),
+				}),
+			},
+			"record_auth_type": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.IntInSlice([]int{
+					int(RecordAuthTypeLink), int(RecordAuthTypeEnterprise), int(RecordAuthTypeAttendee),
+				}),
+			},
+			"participant_number": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+			"participant": {
```

Comment:
all parameters in this structure are optional, is it ok if the user input an empty structure?

Reply:
That‘s no problem, the empty structure will be a nil pointer.